draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5759,6 +5787,37 @@ Expect: 100-continue
    is listed as acceptable, the origin server &SHOULD; send a response
    without any content-coding.
 </t>
+<t>
+   When the Accept-Encoding header field is present in a response, it indicates
+   what content codings the resource was willing to accept in the associated
+   request. The field value is evaluated the same way as in a request.
```

Comment:
"associated request" doesn't seem appropriate here; we use that elsewhere to mean the request in a specific exchange, and so by definition this information can't ever be used (unless we want to get into the exact semantics of what a repeated / retried request is -- but that's another windmill).

I'd suggest something like "... what content codings the resource is currently willing to accept in requests similar to that associated with the response." 

And below: "Note that this information is a hint only; the server's willingness to accept an encoding can be influenced by many factors, including resource identity, time, request method, and other factors."

Reply:
But that's exactly what it means here (the request leading to the response being discussed)...

I'm ok with the rephrased note, but it really doesn't seem to say anything different (the current text in the PR is from <https://greenbytes.de/tech/webdav/rfc7694.html#rfc.section.3.p.3>.

Reply:
I see. It might be good to explicitly state something like "Clients can use this information to optimistically encode future requests. However, because this information is specific to the associated request..."

Reply:
Not sure here. Clients always could use content-codings in requests. All we add here is some signalling from server to client that gives more information. And the paragraph already clearly says what it applies to, and when it could change.

Can you propose some concrete text change?